examples/video/videoGrabberExample/src/ofApp.cpp

```diff
@@ -2,123 +2,105 @@
 
 //--------------------------------------------------------------
 void ofApp::setup(){
-	
-	camWidth 		= 320;	// try to grab at this size. 
-	camHeight 		= 240;
-	
-    //we can now get back a list of devices. 
-	vector<ofVideoDevice> devices = vidGrabber.listDevices();
-	
+    camWidth = 320;  // try to grab at this size.
+    camHeight = 240;
+
+    //we can now get back a list of devices.
+    vector<ofVideoDevice> devices = vidGrabber.listDevices();
+
     for(int i = 0; i < devices.size(); i++){
-		cout << devices[i].id << ": " << devices[i].deviceName; 
-        if( devices[i].bAvailable ){
-            cout << endl;
+        if(cams[i].bAvailable){
```

Comment:
where does cams[i] come from? shouldn't it be devices? 
did you try and compile this before submitting this PR?